### PR TITLE
Change name to CaptureRenderTagHelper

### PR DESCRIPTION
**Pull request by petero-dk**
And update functionality to support the namechange.

### CODE REVIEW

File: CaptureRenderTagHelper.Benchmarks/Benchmarks/ScriptCaptureTagBenchmark.cs [hunk 1]
```diff
@@ -7,13 +7,13 @@
...
-namespace ScriptCaptureTagHelper.Benchmarks.Benchmarks
+namespace CaptureRenderTagHelper.Benchmarks.Benchmarks
 {
     [MemoryDiagnoser]
-    public class ScriptCaptureTagBenchmark
+    public class CaptureRenderTagBenchmark
     {
         private ViewContext _viewContext;
-        private ScriptCaptureTagHelper _captureTag;
+        private CaptureTagHelper _captureTag;
```
Comment (BerserkerDotNet) [inline CaptureRenderTagHelper.Benchmarks/Benchmarks/ScriptCaptureTagBenchmark.cs]: Rename file to be the same as class name.

Comment (BerserkerDotNet) [inline CaptureRenderTagHelper.Benchmarks/Benchmarks/ScriptCaptureTagBenchmark.cs]: Rename file to be the same as class name.
This test is specific to `Capture` functionality, consider naming it `CaptureTagBenchmark`.

File: CaptureRenderTagHelper.Benchmarks/Benchmarks/ScriptRenderTagBenchmark.cs [hunk 2]
```diff
@@ -7,13 +7,13 @@
...
-namespace ScriptCaptureTagHelper.Benchmarks.Benchmarks
+namespace CaptureRenderTagHelper.Benchmarks.Benchmarks
 {
     [MemoryDiagnoser]
     public class ScriptRenderTagBenchmark
     {
         private ViewContext _viewContext;
-        private ScriptRenderTagHelper _renderTag;
+        private RenderTagHelper _renderTag;
         private TagHelperOutput _renderOutput;
```
Comment (BerserkerDotNet) [inline CaptureRenderTagHelper.Benchmarks/Benchmarks/ScriptRenderTagBenchmark.cs]: Rename class and file to `RenderTagBenchmark`

File: CaptureRenderTagHelper.Sample/CaptureRenderTagHelper.Sample.csproj [hunk 3]
```diff
@@ -1,14 +1,14 @@
...
-    <TargetFramework>netcoreapp2.0</TargetFramework>
-  </PropertyGroup>
-  <ItemGroup>
-    <PackageReference Include="Microsoft.AspNetCore.All" Version="2.0.6" />
-  </ItemGroup>
-  <ItemGroup>
-    <ProjectReference Include="..\ScriptCaptureTagHelper\ScriptCaptureTagHelper.csproj" />
-  </ItemGroup>
-  <ItemGroup>
-    <Folder Include="Views\Shared" />
```
Comment (BerserkerDotNet) [inline CaptureRenderTagHelper.Sample/CaptureRenderTagHelper.Sample.csproj:12]: Rename

File: CaptureRenderTagHelper/Extensions/HtmlHelperExtensions.cs [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+using Microsoft.AspNetCore.Mvc.Rendering;
+
+namespace CaptureRenderTagHelper.Extensions
+{
+    public static class HtmlHelperExtensions
+    {
+        public static bool HasCaptured(this IHtmlHelper htmlHelper, string name)
+        {            
+            var key = $"Script_{name}";
+            if (htmlHelper.ViewContext.HttpContext.Items.ContainsKey(key) &&
```
Comment (BerserkerDotNet) [inline CaptureRenderTagHelper/Extensions/HtmlHelperExtensions.cs:11]: nit: Change to `Element`

File: CaptureRenderTagHelper/Types/ContentCapture.cs [hunk 5]
```diff
@@ -0,0 +1,21 @@
+﻿using System.Collections.Generic;
+using Microsoft.AspNetCore.Razor.TagHelpers;
+
+namespace CaptureRenderTagHelper.Types
+{
+    public class ContentCapture
+    {
+        private readonly List<ContentBlock> _scriptBlocks = new List<ContentBlock>();
```
Comment (BerserkerDotNet) [inline CaptureRenderTagHelper/Types/ContentCapture.cs]: nit: _elementBlocks

File: CaptureRenderTagHelper/CaptureTagHelper.cs [hunk 6]
```diff
@@ -1,21 +1,22 @@
...
 using Microsoft.AspNetCore.Mvc.ViewFeatures;
 using Microsoft.AspNetCore.Razor.TagHelpers;
-using ScriptCaptureTagHelper.Types;
+using CaptureRenderTagHelper.Types;
 using System.Linq;
 using System.Threading.Tasks;
 
-namespace ScriptCaptureTagHelper
+namespace CaptureRenderTagHelper
 {
```
Comment (BerserkerDotNet) [inline CaptureRenderTagHelper/CaptureTagHelper.cs:9]: html element

Comment (BerserkerDotNet) [inline CaptureRenderTagHelper/CaptureTagHelper.cs:9]: `html element` here and below

File: CaptureRenderTagHelper/CaptureTagHelper.cs [hunk 7]
```diff
@@ -51,20 +58,20 @@ public override async Task ProcessAsync(TagHelperContext context, TagHelperOutpu
...
             
             if (capture == null)
             {
-                capture = new ScriptCapture();
+                capture = new ContentCapture();
                 ViewContext.HttpContext.Items.Add(key, capture);
             }
             
             var order = Priority ?? int.MaxValue;
-            capture.Add(content, attributes, order, AllowMerge);
```
Comment (BerserkerDotNet) [inline CaptureRenderTagHelper/CaptureTagHelper.cs:67]: nit: `Element`

**Review by BerserkerDotNet — CHANGES_REQUESTED** [review]
Overall looks good, I'd like to run a few tests after you fix build issues. (Some references are not resolving)

File: CaptureRenderTagHelper/CaptureTagHelper.cs (hunk 6 above)
Comment (petero-dk) [reply CaptureRenderTagHelper/CaptureTagHelper.cs:9]: I think this is fixed now, but I am not sure

File: CaptureRenderTagHelper/CaptureTagHelper.cs (hunk 7 above)
Comment (petero-dk) [reply CaptureRenderTagHelper/CaptureTagHelper.cs:67]: I think this is fixed now, but I am not sure

**Comment by petero-dk** [comment]
Did you see the update?

**Comment by BerserkerDotNet** [comment]
Sorry, I was busy for couple of days. I've merged your PR, I'll need to do some housekeeping around build and add more tests and hopefully, by the end of this week, I'll push a new version to NuGet.

Thank you for the idea and implementation, great work!

**Comment by petero-dk** [comment]
Perfect, you're welcome. It will be good to be back on the official package again :-)

**Pull request merged**